### DISCUSSION TITLE
API 요청시 공통파라미터에 대하여

### DISCUSSION THREAD

**Discussion by laigasus** [Strong-Potato/TripVote-BE#50] [General]
https://github.com/Strong-Potato/TripVote-BE/pull/48#discussion_r1447409032 에서 시작된 주제입니다.

도메인 별로 Request param 의 이름이 공통되는 것들에 대해 어떻게 정할지 논의가 필요합니다.

다 다르네요....저만...

![image](https://github.com/Strong-Potato/TripVote-BE/assets/26517061/4b421a7e-5407-4db6-ab78-9e91f9ca2e36)
![image](https://github.com/Strong-Potato/TripVote-BE/assets/26517061/19db51a4-0447-426a-af9e-f1d97faac2ec)
![image](https://github.com/Strong-Potato/TripVote-BE/assets/26517061/e36e0009-fdab-4692-948b-af73f0178f12)

**Comment by NoJaeHyuk** [comment]
공통으로 페이징 관련 파라미터만 맞추면 될까요? 아니면 좀 더 상세히 정해야 할 부분이 있을까요?

**Reply by laigasus** [reply]
당장은 페이징 파라미터만 떠오르네요.
